src/wp-includes/site-previews.php

```diff
@@ -0,0 +1,13 @@
+<?php
```

Comment:
Could this function be added to `src/wp-includes/block-editor.php` in order to avoid creating a new file for only this function?